daprdocs/content/en/operations/monitoring/logging/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Logging"
+linkTitle: "Logging"
+weight: 400
+description: "How to setup loggings for Dapr sidecar, and your application"
+---
```

Comment:
We should add interlinking both to and from the "Troubleshooting --> Logs" document

Reply:
Done